angular/projects/admin-nrpti/src/app/mines/mines-routing.module.ts

```diff
@@ -29,6 +31,42 @@ const routes: Routes = [
         resolve: {
           mines: MinesListResolver
         }
+      },
+      {
+        path: ':mineId',
+        data: {
+          breadcrumb: 'Mine Details'
+        },
+        children: [
+          {
+            path: '',
+            redirectTo: 'detail',
+            pathMatch: 'full'
+          },
+          {
+            path: 'detail',
+            component: MinesDetailComponent,
+            canActivate: [CanActivateGuard],
+            data: {
+              breadcrumb: 'detail'
```

Comment:
This breadcrumb should just be set to null
 ```breadcrumb: null```

Reply:
Updated!